rfc/rfc-0000-move-off-monorepo.md

```diff
@@ -0,0 +1,57 @@
+
+## Motivation
+
+The motivation behind Cartographer as an OSS project aims to build a community and ecosystem to aid supply chain choreography.  Cartographer itself is an "umbrella" for this ecosystem, the first step of this was to build the core components and now we are looking ahead to extend by adding more capabilities.
+ 
+As expressed by maintainers of the https://github.com/vmware-tanzu/cartographer repository it will become hard to manage it if we use it as a monorepo and continue to add more capabilities like conventions etc.
+It is expected that this RFC describes how we split the current repository so it allows the Cartographer OSS project to evolve and scale beyond the concrete capabilities the core aims to provide.
+For example splitting out from the current repo:
+
+-  the __`Runnable`__ feature so it can release independently and install standalone in a target deployment cluster,  avoids deploying all of Cartographer which contains build cluster CRDs not desired in runtime clusters
+- the __examples directory__ so we can build up a wide range of examples that include integrations with wider platform services like conventions.  This also helps encourage contribution of community supply chains for different integrations where we can add automated testing that runs against different setups.
+- the __website__ so we can add documentation for these new integrations and platform capabilities outside of the core repository.  I.e. Documenting services like the conventions or a Jenkins integration should not require a change to the core cartographer repository, this should live outside in its own repository.
+
+## Possible Solutions
+
+There seems to be only two options possible solutions:
+- __monorepo__ we could revisit the [convention proposal](https://github.com/vmware-tanzu/cartographer/pull/514) decision and look at ways to address the monorepo concerns however there seemed a general consensus on the [office hours call](https://www.youtube.com/watch?v=iCURQqV52Uw&t=1173s) that this was not desired.
+- __microservice__ As conventions are now being worked on in a different repository, we should look at splitting out parts that currently reside in https://github.com/vmware-tanzu/cartographer.
+
+Suggestions:
+
+-  __`Runnable`__ feature so it can install standalone in a target deployment cluster and avoids deploying all of Cartographer
+-  __examples directory__ so we can build up a wide range of examples that include integrations with wider platform services like conventions.
+-  __website__ so we can add documentation for these new integrations and platform capabilities outside of the core repository.  I.e. documenting a service like the convention components or a Jenkins integration should not require a change to the core cartographer repository, this should live outside in it's own repository
+- __others?__
```

Comment:
I'm particularly not a fan of having the website layout etc as part of the same codebase as the controllers themselves, but I do find useful the benefits that having documentation in the same repo as the code brings, like the ease of having PRs that update code also updating documentation (naturally, not a huge deal in case we do split it - as long as we ensure that we're always keeping the docs in sync etc).